src/date-picker/README.md

```diff
@@ -12,9 +12,9 @@ Basic initialization
 ```html
 <igx-datePicker></igx-datePicker>
 ```
-Custom formatter function with passed initial date.
+Custom formatter function with passed initial date and locale.
 ```html
-<igx-datePicker [formatter]="customFormatter" [value]="dateValue">
+<igx-datePicker [formatter]="customFormatter" [value]="dateValue" [locale]="'en-US'">
```

Comment:
@Aleksandyr @rkaraivanov I suppose this should also be corrected in the calendar?